support-frontend/assets/pages/digital-subscription-landing/componentsB/digitalSubscriptionLandingHeader.jsx

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { FlashSaleCountdownInHero } from 'components/flashSaleCountdown/flashSaleCountdown';
+import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+
+import SvgChevron from 'components/svgs/chevron';
+import AnchorButton from 'components/button/anchorButton';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { sendTrackingEventsOnClick, type SubscriptionProduct } from 'helpers/subscriptions';
+import { flashSaleIsActive, getSaleCopy } from 'helpers/flashSale';
+import PaymentSelection from 'pages/digital-subscription-landing/components/paymentSelection/paymentSelection';
+
+import ProductPagehero from 'components/productPage/productPageHero/productPageHero';
+
+import { showUpgradeMessage } from '../helpers/upgradePromotion';
+import { showCountdownTimer } from '../../../helpers/flashSale';
+import { CampaignHeader } from '../components/digitalSubscriptionLandingHeader';
+
+import './theMoment.scss';
+
+
+// ----- Types ----- //
+
+type PropTypes = {|
+  countryGroupId: CountryGroupId,
+  dailyEditionsVariant: boolean,
+|};
+
+function getCopy(product: SubscriptionProduct, country: CountryGroupId) {
+  if (showUpgradeMessage()) {
+    return {
+      heading: 'Digital Pack',
+      subHeading: 'Upgrade your subscription to Paper+Digital now',
+    };
+  }
+  if (flashSaleIsActive(product, country)) {
+    const saleCopy = getSaleCopy(product, country);
+    return {
+      heading: `${saleCopy.landingPage.heading}`,
+      subHeading: `${saleCopy.landingPage.subHeading}`,
+    };
+  }
+  return {
+    heading: 'Digital Pack subscriptions',
+    subHeading: 'The premium Guardian experience, ad-free on all your devices',
+  };
+}
+
+const PaymentSelectionContainer = ({ dailyEditionsVariant }: { dailyEditionsVariant: boolean }) => (
+  <div className="payment-selection-container">
+    <LeftMarginSection>
+      <PaymentSelection dailyEditionsVariant={dailyEditionsVariant} pageType="B" />
```

Comment:
The `PaymentSelection` component doesn't use the `dailyEditionsVariant` prop. You should probably change it to use it  and remove `pageType` (this would mean we only have the one way of determining which variant to show) or alternatively remove `dailyEditionsVariant` from the props passed in here (I think this is a less good option).